cmd/yggdrasil/main.go

```diff
@@ -292,10 +292,10 @@ exit:
 }
 
 func (n *node) shutdown() {
-	n.core.Stop()
 	n.admin.Stop()
 	n.multicast.Stop()
 	n.tuntap.Stop()
+	n.core.Stop()
 	os.Exit(0)
```

Comment:
We should get rid of all use of `os.Exit(0)`, just let things return (and run `defer`s) if there's no error.